docs/source/api.rst

```diff
@@ -0,0 +1,212 @@
+
+On the other hand, ``hi_res`` is the better choice for PDFs that may have text within embedded images, or for achieving greater precision of `element types <https://unstructured-io.github.io/unstructured/getting_started.html#document-elements>`_ in the response JSON. Be aware that ``hi_res`` requests may take 20 times longer to process compared to the ``fast`` option. See the example below for making a ``hi_res`` request.
+
+.. code:: shell
+	
+  curl -X 'POST' \
+  'https://api.unstructured.io/general/v0/general' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: multipart/form-data' \
+  -H 'unstructured-api-key: <YOUR API KEY>' \
+  -F 'files=@sample-docs/layout-parser-paper.pdf' \
+  -F 'strategy=hi_res' \
+  | jq -C . | less -R
+
+The ``ocr_only`` strategy runs the document through Tesseract for OCR. Currently, ``hi_res`` has difficulty ordering elements for documents with multiple columns. If you have a document with multiple columns that do not have extractable text, it's recommended that you use the ``ocr_only`` strategy. Please be aware that ``ocr_only`` will fall back to another strategy if Tesseract is not available.
+
+For the best of all worlds, ``auto`` will determine when a page can be extracted using ``fast`` or ``ocr_only`` mode, otherwise, it will fall back to hi_res.
+
+
+XML Tags
+=========
+
+When processing XML documents, set the ``xml_keep_tags`` parameter to ``true`` to retain the XML tags in the output. If not specified, it will simply extract the text from within the tags.
+
+.. code:: shell
+	
+  curl -X 'POST' \
+  'https://api.unstructured.io/general/v0/general' \
+  -H 'accept: application/json'  \
+  -H 'Content-Type: multipart/form-data' \
+  -H 'unstructured-api-key: <YOUR API KEY>' \
+  -F 'files=@sample-docs/fake-xml.xml' \
+  -F 'xml_keep_tags=true' \
+  | jq -C . | less -R
+
+
+Using the API Locally
+**********************
+
+To get started you'll need to fork the ``unstructured-api`` repository `here <https://github.com/Unstructured-IO/unstructured-api>`_.
+
+* Run ``make install``
+* Sart one of the following:
+	- A local jupyter notebook server with ``make run-jupyter``
+	- The fast-API with ``make run-web-app``
+
+NOTE: See the `Unstructured Installation guide <https://unstructured-io.github.io/unstructured/installing.html>`_ for the many OS dependencies that are required, if the ability to process all file types is desired.
+
+You can now hit the API locally at port 8000. The ``sample-docs`` directory has several example file types that are currently supported.
+
+For example:
+
+.. code:: shell
+	
+  curl -X 'POST' \
+  'http://localhost:8000/general/v0/general' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: multipart/form-data' \
+  -F 'files=@sample-docs/family-day.eml' \
+  | jq -C . | less -R
+
+
+Generating Python Files from the Pipeline Notebooks
```

Comment:
I'd drop this section since it's already in the API repo README and generating Python files from the API is a vestige of our old "pipeline notebook" concept. Most users won't need to do that and the ones who do can see it in the repo.